samples/web/content/apprtc/index.html

```diff
@@ -38,6 +38,20 @@
     <video id="remote-video" autoplay></video>
     <video id="local-video" autoplay muted></video>
   </div>
+  
+  <div id="room-selection" class="hidden">
+     <h1>AppRTC</h1>
+     <p id="instructions">Please enter a room name. Room names are shared with everyone, so think of something unique and send it to a friend.</p>
```

Comment:
"Shared with everyone" seems confusing to me. I'd rather go with simply "Please enter a room name" before we get more polished text from UI designer.